templates/design_system/authorize_v2.html

```diff
@@ -27,28 +27,33 @@
 		<h1 class="ds-u-font-size--h2">{{ application.name }} has asked for some of your data.</h1>
 
 		<ul>
-			<li>Information about your doctor/hosiptal visits</li>
+			<li>Information about your doctor/hospital visits</li>
 			<li>Information about the prescription medications you take</li>
-			<li>Personal Information like your name, address, date of birth, race, and gender</li>
+			{% if application.require_demographic_scopes != False %}
+				<li>Personal Information like your name, address, date of birth, race, and gender</li>
+			{% endif %}
+
 		</ul>
 
 		{% if not error %}
 		<form id="authorizationForm" method="post">
-			<fieldset class="ds-c-fieldset bene-auth-fieldset">
-				<legend class="ds-c-label">
-					<h2 class="ds-u-font-size--h3">Privacy Options</h2>
-				</legend>
-				<input class="ds-c-choice" id="radio-1" type="radio" name="block_personal_choice" value="False" checked />
-				<label for="radio-1">
-					<span class="ds-u-font-size--lead">Share all of your data</span>
-					<span class="ds-c-field__hint">This app will have access to both your healthcare data and some personal information</span>
-				</label>
-				<input class="ds-c-choice" id="radio-2" type="radio" name="block_personal_choice" value="True"/>
-				<label for="radio-2">
-					<span class="ds-u-font-size--lead">Share healthcare data, but not your personal info</span>
-					<span class="ds-c-field__hint">Block some of your personal data like name, address, date of birth, race, and gender</span>
-				</label>
-			</fieldset>
+			{% if application.require_demographic_scopes != False %}
```

Comment:
Stylistic choice but I would avoid using `!=` in truth testing to make this simpler.